backend/typeshare.toml

```diff
@@ -1,3 +1,3 @@
 [typescript.type_mappings]
 "NaiveDate" = "Date"
-"BigDecimal" = "number"
\ No newline at end of file
+"BigDecimal" = "number"
```

Comment:
```suggestion
```

Reply:
TypeShare cannot handle some types from 3rd party crates so we have to define it ourselves otherwise we end up with a BigDecimal type in our TypeScript interface which the Frontend has no clue about

Reply:
Yes, this was my suggestion: to not use bigdecimal. At least for our money, pH values etc. it is over-the-top.